### PR TITLE
Add a warning that files are not shared between Storage Foundation and OPFS.

### PR DESCRIPTION
**Pull request by rstz**

### CODE REVIEW

**Review by tomayac — APPROVED** [review]
LGTM

**Review by fivedots — APPROVED** [review]
LGTM, thanks!

File: AccessHandle.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ System](https://docs.google.com/document/d/121OZpRk7bKSF7qU3kQLqAEUVSNxqREnE98ma
 [Recommendation for Augmented
 OPFS](https://docs.google.com/document/d/1g7ZCqZ5NdiU7oqyCpsc2iZ7rRAY1ZXO-9VoG4LfP7fM).
 
+Although this proposal is the successor "in spirit" to Storage Foundation API, the two APIs operate on entirely different sets of files. There exists no way of accessing a file stored through Storage Foundation API using the Origin Private File System, and vice versa.
```
Comment (tomayac) [inline AccessHandle.md:63]:
```suggestion
Although this proposal is the successor "in spirit" to the Storage Foundation API, the two APIs operate on entirely different sets of files. There exists no way of accessing a file stored through the Storage Foundation API using the Origin Private File System, and vice versa.
```

**Review by tomayac — APPROVED** [review]
Two nits relayed from our tech writer.

**Pull request merged**